pennylane/_qubit_device.py

```diff
@@ -769,20 +773,23 @@ def statistics(
               ``shot_range=[35, 135]``, ``bin_size=100``.
         """
         if observables is not None:
-            if isinstance(observables, QuantumScript):
-                circuit = observables
-                measurements = circuit.measurements
-            else:
-                warnings.warn(
-                    message="Using a list of observables in ``QubitDevice.statistics`` is "
-                    "deprecated. Please use a ``QuantumTape`` instead.",
-                    category=UserWarning,
-                )
-                measurements = observables
-        elif circuit is not None:
+            warnings.warn(
+                message="The ``observables`` argument in ``QubitDevice.statistics`` is deprecated. "
+                "Please use ``circuit`` instead.",
+                category=UserWarning,
+            )
```

Comment:
@lillian542 thanks for adding this. One small question -- will this lead to users seeing deprecation warnings that they won't be able to address (due to plugins not being updated)? Or have all plugins/devices already been updated to avoid using this deprecated behaviour?

Reply:
All plugins have been already updated to use `circuit` instead.